docker-app/qfieldcloud/core/views/files_views.py

```diff
@@ -220,8 +237,10 @@ def post(self, request, projectid, filename, format=None):
                 # using the 'X-Request-Id' added to the request by RequestIDMiddleware
                 name=f"{request.META.get('X-Request-Id')}_{projectid}",
                 pre_serialization=request.attached_keys,
-                post_serialization=str(request_attributes),
+                post_serialization=",".join(QfcMultiPartSerializer.errors)
+                + str(request_attributes),
```

Comment:
```suggestion
                post_serialization=(",".join(QfcMultiPartSerializer.errors)
                + str(request_attributes)),
```
Does black accept this? It's super confusing like this. -1 for formatters these days :)

Reply:
Done